cmd/podman/pod_kill.go

```diff
@@ -55,7 +55,7 @@ func podKillCmd(c *cliconfig.PodKillValues) error {
 	}
 	defer runtime.Shutdown(false)
 
-	var killSignal uint = uint(syscall.SIGTERM)
+	var killSignal = uint(syscall.SIGTERM)
```

Comment:
while we're here, I think it's more idiomatic to do 
`killSignal := uint(syscall.SIGTERM)`